src/engine/Server.cpp

```diff
@@ -610,6 +613,8 @@ json Server::composeErrorResponseJson(
 json Server::composeStatsJson() const {
   json result;
   result["name-index"] = index_.getKbName();
+  result["git-hash-index"] = index_.getGitHash();
+  result["git-hash-server"] = gitHash_;
```

Comment:
On a second thought, can't we just use `qlever::version::GitShortHash` here directly?

Reply:
I think you are right. This requires an include of `src/CompilationInfo.h` and I though that should be done only in `ServerMain.cpp` and `IndexBuilderMain.cpp`. But I just realized that it is also included in `src/index/IndexImpl.cpp`, even before this PR.

Reply:
@RobinTF Thanks again for the suggestion. Using `qlever::version::GitShortHash` did not work (linker error), but there is `qlever::version::gitShortHashWithoutLinking` for exactly that purpose.